app/Http/Controllers/TaskController.php

```diff
@@ -145,18 +145,18 @@ public function taskcollection(){
 
     }
 
-    public function sort(Request $request)
```

Comment:
This isn't part of your work but i see that you have made changes to it, why?

Reply:
This was a shift space from what I can see. But I never touched the code.